lite/examples/image_classification/android/README.md

```diff
@@ -17,7 +17,8 @@ These instructions walk you through building and running the demo on an Android
 device. For an explanation of the source, see
 [TensorFlow Lite Android image classification example](EXPLORE_THE_CODE.md).
 
-<!-- TODO(b/124116863): Add app screenshot. -->
+<img src= https://github.com/SunitRoy2703/examples/raw/image-classification-android-screenshot/lite/examples/image_classification/android/screenshot.png width="350" />
```

Comment:
There is no need to add an image.

Please use:
src="https://www.tensorflow.org/lite/examples/image_classification/images/android_banana.png"

Thanks!

Reply:
Thank you for reviewing @lintian06 
I have removed the image and changed the image src to "https://www.tensorflow.org/lite/examples/image_classification/images/android_banana.png"

Reply:
Please surround src with "". Thank you!

Reply:
thanks @lintian06 
I have added image src with "" surrounding it.